lectureNine/lectureNine.md

```diff
@@ -428,4 +427,4 @@ which is a bounty we completed for Agoric.
 
 ### Reserve
 There's an [ongoing discussion](https://github.com/Agoric/agoric-sdk/discussions/6967) about the `Reserve`, 
-we'll update here once our questions are settled.
\ No newline at end of file
+we'll update here once our questions are settled.
```

Comment:
oops. this whitespace-only change is an artifact of using the web UI for making commits.